config/src/iroha.rs

```diff
@@ -240,13 +238,11 @@ mod tests {
                 .gossip_period_ms
                 .expect("Gossip period was None")
         );
-        Ok(())
     }
 
     #[test]
     fn example_json_proxy_builds() -> Result<()> {
-        let cfg_proxy = ConfigurationProxy::from_path(CONFIGURATION_PATH)
-            .wrap_err("Failed to read configuration from example config")?;
+        let cfg_proxy = ConfigurationProxy::from_path(CONFIGURATION_PATH);
         cfg_proxy.build()?;
```

Comment:
```suggestion
        ConfigurationProxy::from_path(CONFIGURATION_PATH).build()?;
```

Reply:
```suggestion
        ConfigurationProxy::from_path(CONFIGURATION_PATH).build().expect()
```
And remove `Ok(())` to conform to the previous test.